### PR TITLE
2.12.1

### PR DESCRIPTION
**Pull request by akravchenko564**

### CODE REVIEW

File: javascript.ru/2.12.1/2.12.1.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title></title>
```
Comment (KasichMax) [inline javascript.ru/2.12.1/2.12.1.html]: Convert spaces to tab.

File: javascript.ru/2.12.1/2.12.1.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿if ("0") {
+	alert('Привет');
+}
```
Comment (KasichMax) [inline javascript.ru/2.12.1/2.12.1.js]: Add another ways to test:
1. if (0) ...
2. if ("") ...
3. if (undefined) ...

Comment (akravchenko564) [reply javascript.ru/2.12.1/2.12.1.js]: Done 62c6e67.

File: javascript.ru/2.12.1/2.12.1.html (hunk 1 above)
Comment (akravchenko564) [reply javascript.ru/2.12.1/2.12.1.html]: Done 62c6e67.

File: javascript.ru/2.12.1/2.12.1.js (hunk 2 above)
Comment (akravchenko564) [reply javascript.ru/2.12.1/2.12.1.js]: Done c251d36.

**Review by KasichMax — APPROVED** [review]

**Pull request merged**